addons/payment/tests/test_flows.py

```diff
@@ -221,7 +222,9 @@ def test_pay_redirect_if_no_partner_exist(self):
 
         # Pay without a partner specified --> redirection to login page
         response = self.portal_pay(**route_values)
-        self.assertTrue(response.url.startswith(self._build_url('/web/login?redirect=')))
+        url = urlparse(response.url)
```

Comment:
Same as above, the error reporting was "False is not True". Now it is something humans can work with.

Reply:
Also side, note. I don't change tests for fun. I **had** that test fail due to errors in my code. I changed the reporting **because** I had to add a `print()` to figure out what was wrong.

Reply:
> Same as above, the error reporting was "False is not True". Now it is something humans can work with.

Yeah, one should at least add a string param to `assertXXX()` so it gives a hint.

Thanks for the explanation :+1: